grovedb/src/lib.rs

```diff
@@ -237,6 +243,82 @@ pub struct GroveDb {
     db: RocksDbStorage,
 }
 
+pub struct s_db_snapshot {
```

Comment:
before it goes from draft to ready-to-review please take a look at https://rust-lang.github.io/api-guidelines/naming.html

Reply:
@fominok Thanks. Will do